packages/MarkdownEditor-Tests.package/MarkdownInlineTextStylerTest.class/instance/assertRuns.values.areStyleOf..st

```diff
@@ -1,6 +1,7 @@
 accessing
-assert: aText hasRuns: runs values: values
+assertRuns: runs values: values areStyleOf: aText
```

Comment:
This is a suggestion which resolves #10. I'm not happy with the naming, but this is the best thing I could have thought of...
When we're happy, we should also use this in the block tests

Reply:
`assertRuns: runs values: values areAttributesOf: aText`? Maybe not this great, too. It lacks the intention that the text is styled by the inlineParser (which should be named inlineStyler in here as well). Why don't you style outside of this method?
Funny: GitHub displays this as MarkdownInlinePARSERTest...

Reply:
Well the intention of this method is to encapsulate everything that is the same for all the tests in which it is used. `privateStyle` always has to be called thus I think it makes sense to have it in the method. Otherwise it would have to be in all tests which would make it more visible but also bulk up the tests. But if we decide that it is more important to see what happens then that would be a good option too.

Reply:
`privateStyle:` is now always called outside of the method. This makes it very visible what is happening.

Reply:
Applying `privateStyle:` outside of the method should work for the inline text styler, but for the block text styler we would either remove the assertion that the string remains unmodified or have a better name which implies that the styler is used...